src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -1301,7 +1308,7 @@ def make_mounted_location(make_random, make_dbfs_data_copy, env_or_skip):
     # external table migration tests
```

Comment:
Note that `make_mounted_location` is different to the other `make_` fixtures as it does not return a `Callable` (using the `factory`). Therefore I can't use a `wait_for_provisioning` field in the signature here. We could refactor it, breaking the use of this fixture.

Reply:
Why it is not a code comment? 😉 it will get lost in PR history